app/controllers/areas_controller.rb

```diff
@@ -1,22 +0,0 @@
-require "mapit_api"
-require "gds_api/exceptions"
-
-class AreasController < ApplicationController
-  def search
-    api_response = fetch_location_for_postcode(params[:postcode])
-    response = MapitApi::AreasByPostcodeResponse.new(api_response)
-    @presenter = AreasPresenter.new(response)
```

Comment:
Can we also remove this presenter and its test?

Reply:
I think we can - let me double-check.

Reply:
Presenter removed.